### ISSUE TITLE
Build warning OOB: BotTestBase.cs(143,26,143,44): warning CS0618: 'FeedbackMiddleware' is obsolete: 'FeedbackMiddleware will no longer work with any VA built with the 0.8 release or newer. For more information, refer to https://aka.ms/bfFeedbackDoc.'

### ISSUE THREAD

**Issue by scheyal** [microsoft/botframework-solutions#3721]
Using latest VSIX in VS 2019 htting this warning out of the box.
Expected: Samples should build cleanly.

```
1>------ Build started: Project: Top, Configuration: Debug Any CPU ------
1>Top -> c:\workspace\botroot\TheAssistant\Top\Top\Top\bin\Debug\netcoreapp3.1\Top.dll
2>------ Build started: Project: Top.Tests, Configuration: Debug Any CPU ------
2>c:\workspace\botroot\TheAssistant\Top\Top\Top.Tests\BotTestBase.cs(143,26,143,44): warning CS0618: 'FeedbackMiddleware' is obsolete: 'FeedbackMiddleware will no longer work with any VA built with the 0.8 release or newer. For more information, refer to https://aka.ms/bfFeedbackDoc.'
2>Top.Tests -> c:\workspace\botroot\TheAssistant\Top\Top\Top.Tests\bin\Debug\netcoreapp3.1\Top.Tests.dll
2>Done building project "Top.Tests.csproj".
```

**Comment by Batta32** [comment]
Hi @scheyal, thanks for reporting this issue. The `FeedbackMiddleware` class is [Obsolete](https://github.com/microsoft/botframework-solutions/blob/master/sdk/csharp/libraries/microsoft.bot.solutions/Feedback/FeedbackMiddleware.cs#L15) and it was removed from the `DefaultAdapter` in the PR #3256.

@peterinnesmsft - should we consider to remove the FeedbackMiddleware implemention from the Bot-Solutions library for C# and TypeScript to avoid these kind of warnings?

**Comment by Batta32** [comment]
This issue is solved with the PR #3733
